.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
```

Comment:
I'm pretty stumped here.  Calling either or both of `atom.packages.loadPackage('linter-haml')` or `atom.packages.loadPackage('language-haml')` in `beforeEach` does not make a difference.

Pretty much the only interesting thing I've been able to glean is that locally, with atom running, if `apm uninstall language-haml` then `apm test`, then on the first run the first couple of tests will fail and the last couple of tests will pass.  Then on the second run they will all pass.  If I have language-haml uninstalled, then they always fail if I do not already have atom running.  

No matter what I try, I _always_ get an error/warning from atom-package-deps.